pkg/connection/reconciler.go

```diff
@@ -108,10 +113,10 @@ func (r *PulsarConnectionReconciler) Reconcile(ctx context.Context) error {
 			}
 			return nil
 		}
-		r.log.Info("Doesn't have unReady resource")
+		r.log.Info("Doesn't have associated unready resource, skip reconcile")
```

Comment:
I think this can be a high level debug log. it doesn't have much useful for users.

Reply:
Sounds better

Reply:
In this part, it can change to `reconcile completed`. But in the sub reconciler, I think it's better to use `skip reconcile`. Skip means it didn't query pulsar, only observe the resource status fields.